src/mautic/index.js

```diff
@@ -68,14 +55,22 @@ export const getContactId = async (email, token) => {
   }
 };
 
-export const subscribeUserToMautic = async (contactId, token) => {
+export const subscribeUserToMautic = async (user, token, cluster, platform) => {
+  const contactId = await getContactId(user.email, token);
   const mauticSubscriptionUrlBase =
     process.env.MAUTIC_SUBSSCRIPTION_API_URL || "";
 
   try {
     const response = await axios.post(
-      `${mauticSubscriptionUrlBase}/segments/contact/add`,
-      suscribeData(contactId),
+      `${mauticSubscriptionUrlBase}/segments/contact/cluster/add`,
```

Comment:
Is there an API specification for Mautic API endpoints?
I think it would be great to check the endpoint specification to match with the changes.